### PR TITLE
Create benefits@ko.yaml

### PR DESCRIPTION
**Pull request by JeongminFelixShin**
Page page location : https://www.ampproject.org/ko/about/benefits/

Three parts of AMP advantages was translated in Korean(Keep your audience engaged, Maximize your revenue, Build for a sustainable future)
But, Remaining three parts has not be translated, so I added Korean translation for those parts (Maintain flexibility and control, Reduce complexity in your operations,Maximize your ROI)

### CODE REVIEW

File: content/learn/about/benefits@ko.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - title@: 잠재고객의 참여 유도하기
+      paragraph@: AMP 페이지는 일반적으로 1초 이내에 로딩됩니다. 즉 타겟 고객이 AMP 페이지에 도달했을 때 원하는 순간에 원하는 정보를 얻게됩니다. 이 결과, 브랜드 메시지에 적극적으로 참여하고,구매를 완료하는 등 원하는 행동을 이끌어낼 수 있습니다.
+      icon: publishers/publisher_icon_fast.svg
+
+    - title@: 수익 극대화하기
+      paragraph@: 웹페이지가 로드되는 데 1초가 더 걸릴 때마다 전환율은 12%씩 떨어집니다. 사람들이 브랜드와 소통하도록 유도하고 캠페인의 판매 및 실적을 늘리고 싶다면 페이지가 로드되는 속도를 높여야 합니다. AMP는 광고, 방문 페이지 또는 전체 웹사이트 등 어디에서나 사용자에게 더 빠른 환경을 제공합니다.
+      icon: adtech/adtech_icon_revenue.svg
+
+    - title@: 유연성과 제어 기능 유지
+      paragraph@: AMP 형식을 채택하면 AMP의 최적화된 웹 구성 요소를 활용하는 동시에 자체 브랜딩 및 툴링을 유지할 수 있습니다. CSS를 사용하여 스타일 지정을 할 수 있고, 동적 데이터를 사용하여 필요한 경우 최신 데이터를 가져올수 있습니다. 또한, A/B 테스트를 통해 고객을 위한 최상의 사용자 경험을 제공할 수 있습니다.
```
Comment (captain-lee) [inline content/learn/about/benefits@ko.yaml]: I think /translations/ko/LC_MESSAGES/messages.po is a correct place to put KO translations for this page (benefits.yaml). For example,
https://github.com/ampproject/docs/blob/5e1de95a7e67664d87ce1e044776d3c3261ca435/translations/ko/LC_MESSAGES/messages.po#L303

File: content/learn/about/benefits@ko.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      icon: card_developers.svg
+
+    - title@: 작업의 복잡성 줄이기
+      paragraph@: AMP 페이지를 만드는 것은 복잡하지않고 간단합니다. 만약 Wordpress 및 Drupal과 같은 널리 사용되는 CMS를 사용하는 경우, 전체 소유한 파일을 며칠 이내에 변환할 수 있습니다. AMP 형식은 쉽게 다룰 수 있으며 AMP 페이지는 사용자가 어떻게 접속하든지 관계없이 일관되게 빠르기 때문에, AMP를 사용하면 각 배포 플랫폼에 맞게 코드를 최적화하는 데 특별한 기술이 필요하거나 걱정할 필요가 없습니다.
+      icon: publishers/publisher_icon_simple.svg
+
+    - title@: ROI 극대화
+      paragraph@: 일반적으로 랜딩 페이지, 광고 또는 웹 컨텐츠를 구축하는 데는 시간이 걸립니다. AMP 페이지는 한 번 제작되어 다양한 배포 플랫폼에 배포될 수 있습니다. AMPHTML 광고로 AMP와 non-AMP 페이지에서 모두 광고를 제공할 수 있습니다. 즉, 광고를 한 번 제작한다면 어디에서나 인상적인 브랜드 경험을 제공할 수 있습니다.
```
Comment (captain-lee) [inline content/learn/about/benefits@ko.yaml]: The correct place would be:
https://github.com/ampproject/docs/blob/5e1de95a7e67664d87ce1e044776d3c3261ca435/translations/ko/LC_MESSAGES/messages.po#L481

File: content/learn/about/benefits@ko.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - title@: 수익 극대화하기
+      paragraph@: 웹페이지가 로드되는 데 1초가 더 걸릴 때마다 전환율은 12%씩 떨어집니다. 사람들이 브랜드와 소통하도록 유도하고 캠페인의 판매 및 실적을 늘리고 싶다면 페이지가 로드되는 속도를 높여야 합니다. AMP는 광고, 방문 페이지 또는 전체 웹사이트 등 어디에서나 사용자에게 더 빠른 환경을 제공합니다.
+      icon: adtech/adtech_icon_revenue.svg
+
+    - title@: 유연성과 제어 기능 유지
+      paragraph@: AMP 형식을 채택하면 AMP의 최적화된 웹 구성 요소를 활용하는 동시에 자체 브랜딩 및 툴링을 유지할 수 있습니다. CSS를 사용하여 스타일 지정을 할 수 있고, 동적 데이터를 사용하여 필요한 경우 최신 데이터를 가져올수 있습니다. 또한, A/B 테스트를 통해 고객을 위한 최상의 사용자 경험을 제공할 수 있습니다.
+      icon: card_developers.svg
+
+    - title@: 작업의 복잡성 줄이기
+      paragraph@: AMP 페이지를 만드는 것은 복잡하지않고 간단합니다. 만약 Wordpress 및 Drupal과 같은 널리 사용되는 CMS를 사용하는 경우, 전체 소유한 파일을 며칠 이내에 변환할 수 있습니다. AMP 형식은 쉽게 다룰 수 있으며 AMP 페이지는 사용자가 어떻게 접속하든지 관계없이 일관되게 빠르기 때문에, AMP를 사용하면 각 배포 플랫폼에 맞게 코드를 최적화하는 데 특별한 기술이 필요하거나 걱정할 필요가 없습니다.
```
Comment (captain-lee) [inline content/learn/about/benefits@ko.yaml]: The correct place would be:
https://github.com/ampproject/docs/blob/5e1de95a7e67664d87ce1e044776d3c3261ca435/translations/ko/LC_MESSAGES/messages.po#L435

File: content/learn/about/benefits@ko.yaml (hunk 2 above)
Comment (JeongminFelixShin) [reply content/learn/about/benefits@ko.yaml]: Thank you for letting me know that.

File: content/learn/about/benefits@ko.yaml (hunk 1 above)
Comment (JeongminFelixShin) [reply content/learn/about/benefits@ko.yaml]: Thank you for letting me know that.

**Comment by JeongminFelixShin** [comment]
wrong location to merge
so it was canceled

**Pull request closed without merging**